src/Faithlife.Data/DbConnectorCommand.cs

```diff
@@ -30,6 +30,12 @@ namespace Faithlife.Data
 		/// </summary>
 		public DbConnector Connector { get; }
 
+		/// <summary>
+		/// The timeout of the command, in seconds.
+		/// </summary>
+		/// <remarks>A value of <c>null</c> will use the database connection's default timeout.</remarks>
+		public int? TimeoutLength { get; }
```

Comment:
I'd rather use `TimeSpan? Timeout` here. Memory usage vs. `int?` isn't significant enough to be a factor in the decision.